src/managers/RoleManager.js

```diff
@@ -115,7 +115,7 @@ class RoleManager extends BaseManager {
   create(options = {}) {
     let { name, color, hoist, permissions, position, mentionable, reason } = options;
     if (color) color = resolveColor(color);
-    if (permissions) permissions = Permissions.resolve(permissions);
+    if (permissions) permissions = Permissions.resolve(permissions).toJSON();
```

Comment:
BigInts don't have a toJSON method.
```suggestion
    if (permissions) permissions = Permissions.resolve(permissions).toString();
```